tests/data_snack/connections/test_redis.py

```diff
@@ -40,11 +58,37 @@ def test_get_many(connection: RedisConnection) -> None:
 
 def test_set_many(connection: RedisConnection) -> None:
     """Testing setting multiple values. Values are provided in a form of a dictionary."""
-    connection.connection.mset.return_value = ["key1", "key2"]
+    connection.connection.set.side_effect = ["key1", "key2"]
+    set_mapping = {"key1": "value1", "key2": "value2"}
+
+    result = connection.set_many(set_mapping)
+    assert set(result) == {"key1", "key2"}
+    assert connection.connection.set.call_count == 2
+    connection.connection.set.assert_has_calls(
+        [call(k, v, ex=None) for k, v in set_mapping.items()]
```

Comment:
Shouldn't ex be 0?

Reply:
Nope, this connection has a parameter ex set to `None` for no expire time, see https://redis-py.readthedocs.io/en/stable/commands.html?highlight=set_many#redis.commands.core.CoreCommands.set